ISL.Providers.Storages.Abstractions.Tests.Unit/StorageAbstractionProviderTests.Exceptions.RemoveAccessPoliciesFromContainerAsync.cs

```diff
@@ -0,0 +1,221 @@
+                await Assert.ThrowsAsync<StorageProviderDependencyException>(
+                    testCode: removeAccessPoliciesFromContainerAsyncTask.AsTask);
+
+            // then
+            actualStorageDependencyProviderException.Should().BeEquivalentTo(
+                expectedStorageDependencyProviderException);
+
+            this.storageProviderMock.Verify(provider =>
+                provider.RemoveAccessPoliciesFromContainerAsync(It.IsAny<string>()),
+                    Times.Once);
+
+            this.storageProviderMock.VerifyNoOtherCalls();
+        }
+
+        [Fact]
+        public async Task
+            ShouldThrowServiceExceptionOnRemoveAccessPoliciesFromContainerAsyncWhenTypeIStorageServiceException()
+        {
+            // given
+            var someException = new Xeption();
+
+            var someStorageValidationException =
+                new SomeStorageServiceException(
+                    message: "Some storage provider service exception occurred",
+                    innerException: someException);
+
+            StorageProviderServiceException expectedStorageServiceProviderException =
+                new StorageProviderServiceException(
+                    message: "Storage provider service error occurred, contact support.",
+                    innerException: someStorageValidationException);
+
+            this.storageProviderMock.Setup(provider =>
+                provider.RemoveAccessPoliciesFromContainerAsync(It.IsAny<string>()))
+                    .ThrowsAsync(someStorageValidationException);
+
+            // when
+            ValueTask removeAccessPoliciesFromContainerAsyncTask =
+                this.storageAbstractionProvider
+                    .RemoveAccessPoliciesFromContainerAsync(It.IsAny<string>());
+
+            StorageProviderServiceException actualStorageServiceProviderException =
+                await Assert.ThrowsAsync<StorageProviderServiceException>(
+                    testCode: removeAccessPoliciesFromContainerAsyncTask.AsTask);
+
+            // then
+            actualStorageServiceProviderException.Should().BeEquivalentTo(
+                expectedStorageServiceProviderException);
+
+            this.storageProviderMock.Verify(provider =>
+                provider.RemoveAccessPoliciesFromContainerAsync(It.IsAny<string>()),
+                    Times.Once);
+
+            this.storageProviderMock.VerifyNoOtherCalls();
+        }
+
+        [Fact]
+        public async Task
+            ShouldThrowUncatagorizedServiceExceptionOnRemoveAccessPoliciesFromContainerAsyncWhenTypeIsNotExpected()
+        {
+            // given
+            var someException = new Xeption();
+
+            var uncatagorizedStorageProviderException =
+                new UncatagorizedStorageProviderException(
+                    message: "Storage provider not properly implemented. Uncatagorized errors found, " +
+                            "contact the storage provider owner for support.",
```

Comment:
Christo pointed out to me earlier that these should only be single indentation
```suggestion
                        "contact the storage provider owner for support.",
```

Reply:
Change pushed